tests/test_metrics_calculate.py

```
@@ -0,0 +1,107 @@
+import pytest
+from app.core.metric_cluster.metrics_calculate import MetricsCalculate, PageRank, Betweenness
+from app.database import neo4j_connection
+
+
+class TestMetricsCalculate:
+    """Тесты для классов расчёта метрик графа."""
+
+    def test_pagerank_initialization(self):
+        """Проверяет инициализацию PageRank с правильными параметрами."""
```

Ruff (0.14.8)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)